app/src/connectors.ts

```diff
@@ -10,3 +17,9 @@ export const WalletConnect = new Connectors.WalletConnectConnector({
   },
   defaultNetwork: 4,
 })
+
+const connector = (process.env.REACT_APP_CONNECTOR || 'MetaMask') as ConnectorsEnum
+
+const connectorToExport = connector === ConnectorsEnum.MetaMask ? { MetaMask } : { WalletConnect }
```

Comment:
PP: I don't love the idea of exporting something based on an environment variable. I think it would be better to export all connectors (as before) and to choose which one to use based on the env var.

Reply:
Done!